support/releaseinfo/releaseinfo_test.go

```diff
@@ -0,0 +1,22 @@
+package releaseinfo
+
+import (
+	"testing"
+
+	"github.com/openshift/hypershift/support/releaseinfo/fixtures"
+)
+
+// TestReleaseInfoPowerVS test validates the presence of the powervs images in the 4.10 release
+func TestReleaseInfoPowerVS(t *testing.T) {
+	metadata, err := DeserializeImageMetadata(fixtures.CoreOSBootImagesYAML_4_10)
+	if err != nil {
+		t.Fatal(err)
+	}
+	arch, ok := metadata.Architectures["ppc64le"]
+	if !ok {
+		t.Fatal("metadata does not contain the ppc64le architecture")
+	}
+	if len(arch.Images.PowerVS.Regions) == 0 {
```

Comment:
type CoreOSPowerVSImage struct {
	Release string `json:"release"`
	Object  string `json:"object"`
	Bucket  string `json:"bucket"`
	URL     string `json:"url"`
}

do we know if any of these are being properly populated?

Reply:
@enxebre amended the latest commit with that additional check as well, ptal